src/CodeViewer/Viewer.cpp

```diff
@@ -198,4 +355,21 @@ void Viewer::HighlightCurrentLine() {
   setExtraSelections({selection});
 }
 
+int Viewer::WidthPercentageColumn() const {
```

Comment:
This (and the next two functions) can be also static, right?

Reply:
Yes, I need to add "static" to my mental checking list

Reply:
AFAIK, they need to be member function because the access `fontMetrics()`.

Reply:
Oh, sorry. Haven't noticed this one. Thanks!

Reply:
Yes, Henning is right. Sorry for my comment and thanks Henning!